Integrations/ThreatQ/ThreatQ.py

```diff
@@ -0,0 +1,1248 @@
+
+SERVER_URL = demisto.params()['serverUrl']
+API_URL = SERVER_URL + '/api'
+API_TOKEN_URL = API_URL + '/token'
+CLIENT_ID = demisto.params()['client_id']
+EMAIL = demisto.getParam('credentials').get('identifier')
+PASSWORD = demisto.getParam('credentials').get('password')
+USE_SSL = not demisto.params().get('insecure', False)
+THRESHOLD = demisto.params().get('threshold')
+if THRESHOLD:
+    THRESHOLD = int(THRESHOLD)
+
+REGEX_MAP = {
+    'email': re.compile(emailRegex, regexFlags),
+    'url': re.compile(urlRegex, regexFlags),
+    'md5': re.compile(r'\b[0-9a-fA-F]{32}\b', regexFlags),
+    'sha1': re.compile(r'\b[0-9a-fA-F]{40}\b', regexFlags),
+    'sha256': re.compile(r'\b[0-9a-fA-F]{64}\b', regexFlags),
+    'domain': re.compile(domainRegex, regexFlags)
+}
+
+TQ_TO_DEMISTO_IOC_TYPES = {
+    'IP Address': 'ip',
+    'IPv6 Address': 'ip',
+    'Email Address': 'email',
+    'URL': 'url',
+    'MD5': 'file',
+    'SHA-1': 'file',
+    'SHA-256': 'file',
+    'FQDN': 'domain'
+}
+
+STATUS_ID_TO_STATUS = {
+    1: 'Active',
+    2: 'Expired',
+    3: 'Indirect',
+    4: 'Review',
+    5: 'Whitelisted'
+}
+
+TYPE_ID_TO_IOC_TYPE = {
+    1: 'Binary String',
+    2: 'CIDR Block',
+    3: 'CVE',
+    4: 'Email Address',
+    5: 'Email Attachment',
+    6: 'Email Subject',
+    7: 'File Mapping',
+    8: 'File Path',
+    9: 'Filename',
+    10: 'FQDN',
+    11: 'Fuzzy Hash',
+    12: 'GOST Hash',
+    13: 'Hash ION',
+    14: 'IP Address',
+    15: 'IPv6 Address',
+    16: 'MD5',
+    17: 'Mutex',
+    18: 'Password',
+    19: 'Registry Key',
+    20: 'Service Name',
+    21: 'SHA-1',
+    22: 'SHA-256',
+    23: 'SHA-384',
+    24: 'SHA-512',
+    25: 'String',
+    26: 'x509 Serial',
+    27: 'x509 Subject',
+    28: 'URL',
+    29: 'URL Path',
+    30: 'User-agent',
+    31: 'Username',
+    32: 'X-Mailer'
+}
+
+TYPE_ID_TO_EVENT_TYPE = {
+    1: 'Spearphish',
+    2: 'Watering Hole',
+    3: 'SQL Injection Attack',
+    4: 'DoS Attack',
+    5: 'Malware',
+    6: 'Watchlist',
+    7: 'Command and Control',
+    8: 'Anonymization',
+    9: 'Exfiltration',
+    10: 'Host Characteristics',
+    11: 'Compromised PKI Certificate',
+    12: 'Login Compromise',
+    13: 'Incident'
+}
+
+TYPE_ID_TO_FILE_TYPE = {
+    1: 'Cuckoo',
+    2: 'CrowdStrike Intelligence',
+    3: 'Early Warning and Indicator Notice (EWIN)',
+    4: 'FireEye Analysis',
+    5: 'FBI FLASH',
+    6: 'Generic Text',
+    7: 'Intelligence Whitepaper',
+    8: 'iSight Report',
+    9: 'iSight ThreatScape Intelligence Report',
+    10: 'IB',
+    11: 'AEC',
+    12: 'Malware Analysis Report',
+    13: 'Malware Initial Findings Report (MFIR)',
+    14: 'Malware Sample',
+    15: 'Packet Capture',
+    16: 'Palo Alto Networks WildFire XML',
+    17: 'PCAP',
+    18: 'PDF',
+    19: 'Private Industry Notification (PIN)',
+    20: 'Spearphish Attachment',
+    21: 'STIX',
+    22: 'ThreatAnalyzer Analysis',
+    23: 'ThreatQ CSV File',
+    24: 'Whitepaper'
+}
+
+HEADERS = {
```

Comment:
Confusing name, might be mistaken with http headers
```suggestion
TABLE_HEADERS = {
```